src/Controllers/Profile/edit.php

```diff
@@ -35,7 +35,8 @@
         ->setEmail($data['email'])
         ->setReceiveEmail($data['receive_email'])
         ->setEduroam($data['eduroam'])
-        ->setBio($data['bio']);
+        ->setBio($data['bio'])
+        ->setContractSigned($data['contract']);
```

Comment:
Since there's situations where this value isn't in the form, this'll throw a notice and try and set it to null